resilience4j-ratpack/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
     compileOnly(libraries.ratpack)
     compile project(':resilience4j-annotations')
     compile project(':resilience4j-framework-common')
+    compileOnly(libraries.micrometer)
```

Comment:
Not sure why `resilience4j-micrometer` is not bringing micormeter core transitively. had to add it explicitly